.istanbul.yml

```diff
@@ -2,7 +2,7 @@ verbose: false
 instrumentation:
     root: .
     default-excludes: true
-    excludes: []
+    excludes: ['**/website/**']
```

Comment:
why we exclude this?

Reply:
We don't handle `jsx` when we run the test. For now, we skip the test on the docs.